notebooks/Falsification.ipynb

```diff
@@ -0,0 +1,887 @@
+{
```

Comment:
@JosephCottam is this notebook just for exploration? I think, if so it should be removed.

Reply:
If you want to write more thorough user documentation, the best place to put it is in the module-level docstring or in the `docs/source/` folder

Reply:
I think it would be better to write these up as documentation instead of notebooks - the problem is jupyter notebooks can get out of date very quickly and are not readily able to be combined with other documentation

Reply:
Falsification is actually a down-stream of conditional independency testing (issue #23).  I don't think its ready for a fully-fledged example yet. If you prefer not to use a jupyter notebook example, then I'm fine with it being removed.

Reply:
Let's just remove it for now, then